avm/res/net-app/net-app-account/capacity-pool/volume/main.bicep

```
@@ -101,6 +101,12 @@ param smbNonBrowsable string = 'Disabled'
 @description('Optional. Define if a volume is KerberosEnabled.')
 param kerberosEnabled bool = false
 
+@description('Optional. Defines the security style of the Volume.')
+param securityStyle string = ''
+
+@description('Optional. Unix Permissions for NFS volume.')
+param unixPermissions string = ''
```

```suggestion
param securityStyle string?

@description('Optional. Unix Permissions for NFS volume.')
param unixPermissions string?
```

If declared as `nullable` you can save yourself the check below and change e.g. `securityStyle: !empty(securityStyle) ? securityStyle : null` to `securityStyle: securityStyle`

---

Thank you Alex, I have made the changes and updated the version